lib/run_cucumber.js

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env node
+
+const fs = require('fs');
+const log = require('./utils/log');
+const session_vars = require('./utils/session_vars');
+
+// import {loadConfiguration, loadSupport, runCucumber } from '@cucumber/cucumber/api';
+const cucumber_api = require('@cucumber/cucumber/api');
+
+(async function main() {
+  let argv = process.argv;
+
+  let just_args = argv.slice(2);
+
+  const environment = { cwd: process.cwd() };
+  const provided_config = {
+    "publishQuiet": true,
+    "require": "./lib/index.js", 
+    "format": ["progress", "summary:cucumber-report.txt"],
+    "paths": ["docassemble/*/data/sources/*.feature"],
+    "retry": 1,
+  }
+  if (just_args.length > 0) {
+    // Cucumber expects tags in boolean expressions, like (@a0 or @a1) and @random.
+    // If we get a list of only tags, then "or" them all together
+    if (just_args.every((val) => val.startsWith("@"))) {
+      provided_config.tags = just_args.join(' or ');
+    }
+    // If we get a list of seemingly no tags, assume they meant to use tags
+    else if (just_args.every((val) => !val.startsWith("@"))) {
+      provided_config.tags = just_args.map((val) => "@" + val).join(' or ');
+    }
+    // If it's a mix of tags and no tags, it's probably a boolean expression!
+    else {
+      provided_config.tags = just_args.join(' ');
+    }
+  }
+  const { runConfiguration } = await cucumber_api.loadConfiguration({
+    provided: provided_config
+  }, environment);
+  var path = require('path');
+  var codeDir = path.dirname(require.main.filename);
+  const support = await cucumber_api.loadSupport({ support: {
+    requireModules: [`${ codeDir }/index.js`],
+    requirePaths: [ `${ codeDir }/index.js`],
+    importPaths: [],
+  }, sources: {paths: []}}, environment);
+
+  const { success } = await cucumber_api.runCucumber({...runConfiguration, support});
+  console.log("%o", success);
```

Comment:
What does "%o" mean?

Reply:
https://developer.mozilla.org/en-US/docs/Web/API/console#outputting_text_to_the_console

Reply:
If we include, we'd add more details, but maybe we don't need it. Maybe more logs are better, though.

Reply:
Can we somehow log the full command that is constructed through this script?

Reply:
We can print the full run configuration and support. Let's add that.